schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/DataIntakeRD.avdl

```diff
@@ -0,0 +1,187 @@
+@namespace("org.gel.models.cva.avro")
+
+/**
+DataIntakeRDProtocol defines three records for injecting variant data
+
+   - TieredVariantInjectRD
+   - ReportedVariantInjectRD
+   - CandidateVariantInjectRD
+
+Every ReportedVariant is uniquely identified by:
+
+   - report event id
+   - report model version
+   - id
+   - version
+   - family id
+   - cohort id
+   - variants identifiers (being a variant identifier formed by chromosome + position + reference + alternate)
+
+Any duplication of a ReportedVariant will be rejected.
+*/
+protocol DataIntakeRDProtocol {
+
+    // CVA references
+    import idl "Variant.avdl";
+    //
+    import idl "../../org.gel.models.report.avro/4.2.0-SNAPSHOT/InterpretationRequestRD.avdl";
+    import idl "../../org.gel.models.report.avro/4.2.0-SNAPSHOT/InterpretedGenomesRD.avdl";
+    import idl "../../org.gel.models.report.avro/4.2.0-SNAPSHOT/ClinicalReportRD.avdl";
+
+    /**
+    Record for tiered variant injection as part of the data intake for CVA
+    */
+    record TieredVariantInjectRD {
+        /**
+        Report avro models version
+        */
+        string reportModelVersion;
+        /**
+        The identifier for the entity containing this reported variant.
+        */
+        string id;
+        /**
+        The version for the entity containing this reported variant. This is a correlative number being the highest
+        value the latest version.
+        */
+        int version;
+        /**
+        The family identifier
+        */
+        string familyId;
```

Comment:
shall we change this to `groupId` to reflect the changes to `group_id` in the CIP API ?